compiler/test/stdlib/number.test.gr

```diff
@@ -6,6 +6,9 @@ import Float32 from "float32"
 import Float64 from "float64"
 import BI from "bigint"
 
+// Constants
+let infinity = 1.0 / 0.0
+let nan = 0.0 / 0.0
```

Comment:
If we decide to export the constants from stdlib, then you should use those instead of redefining here